### PR TITLE
Create Covid 19 Awarness Website

### PR DESCRIPTION
**Pull request by Sulagna-Dutta-Roy**
### Description

(Describe your project over here.)
This is the Covid 19 awareness website. It has 4 different section(home,spread,protection etc). I made it using only html5,css,javascript.You will find donate section for contributrion , I used Razorpay for this. Fully responsive website with support and encourage to everyone. Used two links for covid-19 updated news.

### Checklist

- [x] I've been assigned an issue related to this PR.
- [ ] I've used beautifiers.
- [x] I've added my Project's name and description to `Index.md`
- [x] I've made a README.md file for my Project.
- [x] The README.md file of my project contains Project title, Description, Use of project, Set up, Stack used and Output (Screenshots). 

## Related Issues or Pull Requests number
#187 
(Write your answer here.)

### CODE REVIEW

**Comment by chicken-biryani** [comment]
@Sulagna-Dutta-Roy please edit the PR comment. Describe the project and use "#issue no." Refer to the issue it deals with.

**Comment by chicken-biryani** [comment]
@Sulagna-Dutta-Roy please go through the changes you've done again.

You've changed the files you should not be changing.

Refer to this PR : 

https://github.com/Ayushparikh-code/Web-dev-mini-projects/pull/173

File: README.md [hunk 1]
```diff
@@ -1,141 +1,23 @@
...
-
-```terminal
-cd Web-dev-mini-projects
-```
-
-**4.**  MAKE A NEW FOLDER WITH YOUR PROJECT NAME INSIDE web-dev-mini-projects & Add your project files (eg: index.html ,style.css, script.js) inside that folder
-<br>
-
-**5.**  Also Add a README file in your project folder which consists of Description/screenshots about your project !
-
```
Comment (harshita2216) [inline README.md:42]: You've removed the readme of the repository. You have to create a readme for your own project. @Sulagna-Dutta-Roy

**Review by harshita2216 — CHANGES_REQUESTED** [review]
Also add your project details only in the index.md file.

**Review by Ayushparikh-code — CHANGES_REQUESTED** [review]
Hey @Sulagna-Dutta-Roy you directly added your project files in the original repo, so do make a seperate folder for your project inside repo and then add all files in that folders ...

**Comment by harshita2216** [comment]
@Sulagna-Dutta-Roy, let me know when you're done making the changes.

**Comment by Sulagna-Dutta-Roy** [comment]
I've done done changes. Please check it

File: README.md (hunk 1 above)
Comment (harshita2216) [reply README.md:42]: @Sulagna-Dutta-Roy, Don't remove this, please,

**Comment by harshita2216** [comment]
> I've done done changes. Please check it

Hi, you've deleted the entire readme of the repo. Please check and make the changes. Thanks!

**Comment by Sulagna-Dutta-Roy** [comment]
I've added the readme.md .Please check

**Pull request merged**